Ed-Fi-Alliance-OSS/Ed-Fi-Analytics-Middle-Tier
# [BIA-898] - Adds support for Base collection with Data Standard 3.3.

To test this:
1. Checkout BIA-898 branch.
2. Compile the Solution.
3. Select the tests associated to the views on the Base collection. 
4. Execute these tests
5. All the tests should be executed successfully.
